samples/server/go-server/api/server/api/api.go

```diff
@@ -45,11 +46,18 @@ type Config struct {
 
 // Run initializes grpc and grpc gateway api services on the same address
 func Run(config *Config, storage *server.Storager) {
-	l, err := net.Listen("tcp", config.Address)
+	network, address := "tcp", config.Address
+	if strings.Contains(config.Address, "unix://") || strings.Contains(config.Address, "/") {
```

Comment:
Maybe strings.HasPrefix instead, since it should always start with "unix://" right?

Also, not sure we want to assume unix socket if we find a "/" in general, since you could have something like localhost:8080/api as a TCP endpoint.

Reply:
Makes sense @vtsao. Let me know if you think we should completely removes the `/` prefix.

Reply:
I think we should remove the or check, so remove this:
|| strings.HasPrefix(config.Address, "/")

Reply:
Sure @vtsao makes sense. I removed the second check and updated the code.